.github/jobs/linux.yml

```diff
@@ -31,6 +31,10 @@ jobs:
         cmake .. -G Ninja -D JAVASCRIPTCORE_LIBRARY=/usr/lib/x86_64-linux-gnu/libjavascriptcoregtk-4.0.so -D NAPI_JAVASCRIPT_ENGINE=${{ parameters.JSEngine }} -D CMAKE_BUILD_TYPE=RelWithDebInfo -D BX_CONFIG_DEBUG=ON -D CMAKE_UNITY_BUILD=$(UNITY_BUILD)
```

Comment:
nit: We should stop doing this `mkdir` thing and use `-B` on cmake instead.

Reply:
indeed! I'm doing the change!